pkg/util/podman/db_client.go

```diff
@@ -140,9 +143,18 @@ func (client *DBClient) GetAllContainers() ([]Container, error) {
 }
 
 func (client *DBClient) getDBCon() (*bolt.DB, error) {
-	db, err := bolt.Open(client.DBPath, 0600, nil)
+	dbOptions := bolt.DefaultOptions
```

Comment:
The change looks good to me :+1: but I was wondering if it would it be useful to include a comment to remember that we've modified this function https://github.com/DataDog/datadog-agent/pull/15952/files#diff-b0d639d01df1d6348ddcb6ae2db9e40359fb8483a64f4b7e0a2af5458336ef5cR22 given that most of this code comes from the podman lib.